app/templates/buyers/brief_overview.html

```diff
@@ -38,7 +38,28 @@
           </form>
         </div>
       {% endif %}
-   {% endblock %}
+    {% endblock %}
+
+    {% with messages = get_flashed_messages(with_categories=true) %}
```

Comment:
Could the flash banner go in the `{% block before_heading %}` above, alongside the `delete_requested` banner? (Genuine question - I'm not sure how strict we are about what goes in content blocks)

Reply:
We've also got an almost identical bit of code in the `dashboard.html` template. Any way we could make this more DRY for future re-use? ('No' is an acceptable answer to this question!)

Reply:
In terms of the DRY I'm going to call it as out of scope of this ticket as we've got 8 other pretty much copy-paste occurrences of this pattern but I'll add a ticket to the tech debt board to get that cleaned up. Hopefully someone can find a nice solution and tie it into the base templates.

[messages-repetition-search.txt](https://github.com/alphagov/digitalmarketplace-briefs-frontend/files/1291712/messages-repetition-search.txt)

https://trello.com/c/fLiJa6tD/177-tidy-up-banner-display-code-and-allow-it-to-properly-accept-and-display-flash-messages